### PR TITLE
feat: [ACI-1031] Upgrade requirements to upstream version

### PR DESCRIPTION
**Pull request by KyryloKireiev**
**Description**
Upgrade requirements to current upstream's versions.

**YouTrack**
https://youtrack.raccoongang.com/issue/ACI-1031/Review-PRs-comments

### CODE REVIEW

File: requirements/all.txt [hunk 1]
```diff
@@ -304,7 +314,7 @@ edx-event-bus-kafka==5.7.0
     # via
     #   -r requirements/dev.txt
     #   -r requirements/production.txt
-edx-event-bus-redis @ git+https://github.com/openedx/event-bus-redis.git@v0.5.0
```
Comment (wowkalucky) [inline requirements/all.txt:307]: This downgrade is strange.

File: requirements/common_constraints.txt [hunk 2]
```diff
@@ -3,6 +3,16 @@
 # See BOM-2721 for more details.
 # Below is the copied and edited version of common_constraints
 
+# This is a temporary solution to override the real common_constraints.txt
```
Comment (wowkalucky) [inline requirements/common_constraints.txt:6]: Pls remove this diff manually.

**Review by wowkalucky — APPROVED** [review]

File: requirements/all.txt (hunk 1 above)
Comment (KyryloKireiev) [reply requirements/all.txt]: Fixed

File: requirements/common_constraints.txt (hunk 2 above)
Comment (KyryloKireiev) [reply requirements/common_constraints.txt]: Removed

**Review by wowkalucky — APPROVED** [review]

**Pull request merged**